inst/templates/ad_adpp.R

```diff
@@ -56,27 +56,15 @@ param_lookup <- tibble::tribble(
   "RENALCL", "RENALCL", "CLR", 22
 )
 
-# ASSIGN AVALCAT1
-avalcat_lookup <- tibble::tribble(
-  ~PARAMCD, ~AVALCA1N, ~AVALCAT1,
-  "AUCALL", 1, "< 19",
-  "AUCALL", 2, ">= 19"
+# Assign AVALCATx
+avalcax_lookup <- exprs(
+  ~PARAMCD, ~condition, ~AVALCAT1, ~AVALCA1N,
+  "AUCALL", AVAL < 19, "<19", 1,
+  "AUCALL", AVAL >= 19, ">=19", 2
 )
 
 attr(param_lookup$PPTESTCD, "label") <- "Parameter Short Name"
```

Comment:
Can we update the labels in `pharmaversesdtm` @manciniedoardo ?

Reply:
Haha, should I make a separate issue? :)